client/src/styles/global.scss

```diff
@@ -329,3 +331,89 @@ $primary-color: #112f4e;
     }
   }
 }
+
+
+/* the > is for child only syntax in css. This is required so that lists that are nested under the
+      list are not affected (e.g. <ul><li><p><ul><li>not styled</li></ul></p></li><ul>*/
+.j40-process-list-wrapper > {
+  $j40-steps-list-color: #002D3F;
+  ul {
+    margin-top: 3rem;
+    margin-bottom: 1rem;
+    padding-inline-start: 0;
+
+    > li {
```

Comment:
This `>` syntax means to only apply the style one child element deep. This is required because there are `ul` lists _inside_ the content of these items (see `Data inputs` section).

Bonus, it prevent styles from bleeding down into child elements.